{{ cookiecutter.package_name }}/setup.cfg

```diff
@@ -11,6 +11,7 @@ url = {{ cookiecutter.project_url }}
 edit_on_github = {{ cookiecutter.edit_on_github_extension }}
 github_project = {{ cookiecutter.github_project }}
 python_requires = ">={{ cookiecutter.minimum_python_version }}"
+tests_require = pytest; pytest-astropy; pytest-astropy-header
```

Comment:
should pytest-astropy be a strict dependency?

Reply:
It will be with this patch because I removed the `if ... else ...` check, which I think is more complicated than it's worth.

Reply:
yeah, but I think using pytest-astropy-header should just work for everything, isn't it (at least that was the reason I was pushing for having it as a separate plugin rather than shuffled into pytest-astropy)

Reply:
Correct. Theoretically it would work even if you have older astropy version, but that also makes it a compulsory dependency for tests. Or did I misunderstand your question?

Reply:
no I think we're on the same page. But then  we don't need to list pytest-astropy here, right?

Reply:
Oh yeah...